lib/tokens/tokens.go

```diff
@@ -0,0 +1,42 @@
+package tokens
+
+type Token string
+
+const (
+	OMNI Token = "OMNI"
+	ETH  Token = "ETH"
+)
+
+var (
+	coingeckoIDs = map[Token]string{
+		OMNI: "omni-network",
+		ETH:  "ethereum",
+	}
+)
+
+func (t Token) String() string {
+	return string(t)
+}
+
+func (t Token) CoingeckoID() string {
+	return coingeckoIDs[t]
+}
+
+func FromCoingeckoID(id string) (Token, bool) {
```

Comment:
Is it possible that in future we use something other than **Coingecko**?

Reply:
Yes. Tho I have a feeling ids will be the same. Will deal with it then